.github/workflows/push-sls-image.yml

```diff
@@ -26,10 +27,10 @@ jobs:
         echo "Building image with serverless version: ${{ github.event.inputs.sls }} and node version ${{ github.event.inputs.node-version }}"
```

Comment:
do we have a sls default version?

Reply:
yes, when building the image through push-sis-image GitHub action:

```
on: 
  workflow_dispatch:
    inputs:
      node-version:
        description: 'Node version'
        required: false
        default: '16'
        type: choice
        options:
          - 12
          - 16
          - 18
          - 20
      sls:
        description: 'Serverless version'
        required: false
        default: '3.26.0'
```